traitsui/undo.py

```diff
@@ -72,11 +72,14 @@ class UndoItem(AbstractUndoItem):
     # -------------------------------------------------------------------------
 
     #: Object the change occurred on
-    object = Trait(HasTraits)
+    object = Instance(HasTraits)
+
     #: Name of the trait that changed
     name = Str
+
     #: Old value of the changed trait
     old_value = Property
```

Comment:
change these ones from `Property` to `Property()` too? (Or do that in a separate PR?)

Reply:
I might do that.  I added `()` to a lot of traits as part of the equivalent Pyface PR, but I can't remember how I did it reliably.  Probably just looking for the most common trait types.

Reply:
On second thoughts, not in this PR - it touches too much code.